samples/consumer-driven-contracts/Core_9/Consumer1/Program.cs

```diff
@@ -1,25 +1,29 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
 using NServiceBus;
 
 class Program
 {
-    static async Task Main()
+    public static async Task Main(string[] args)
     {
-        Console.Title = "Consumer1";
-        var endpointConfiguration = new EndpointConfiguration("Samples.ConsumerDrivenContracts.Consumer1");
-        var transport = endpointConfiguration.UseTransport(new LearningTransport());
-        endpointConfiguration.UseSerialization<SystemJsonSerializer>();
-
-        endpointConfiguration.SendFailedMessagesTo("error");
-        endpointConfiguration.EnableInstallers();
-
-        var endpointInstance = await Endpoint.Start(endpointConfiguration);
-
-        Console.WriteLine("Press any key to exit");
+        await CreateHostBuilder(args).Build().RunAsync();
+    }
 
-        Console.ReadKey();
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureServices((hostContext, services) =>
+            {
+                Console.Title = "Server";
```

Comment:
This is a duplicate of line `20`. Can we move the title setup to the `Main` method?